src/flask/globals.py

```diff
@@ -49,3 +49,5 @@
 session: SessionMixin = LocalProxy(  # type: ignore[assignment]
     _cv_request, "session", unbound_message=_no_req_msg
 )
+
+_false_attributes: list[str] = ["0", "false", "no", "null", "none"]
```

Comment:
This is a unmarked behavior change, this list is more exhaustive,

This might create trouble for slightly sarcastic deployments